modules/exploits/windows/smb/smb_relay.rb

```diff
@@ -155,6 +155,12 @@ def available_actions
     actions
   end
 
+  # Called when the module is set as the currently active module
+  def activate
+    super
+    add_info('New in Metasploit 6.4 - The %SMB_SESSION%clr action within this module can open an interactive session')
```

Comment:
```suggestion
    add_info('New in Metasploit 6.4 - The %CREATE_SMB_SESSION%clr action within this module can open an interactive session')
```

Maybe since we're here we can verb-ify the action name too 😄